test/mailers/business_mailer_test.rb

```diff
@@ -0,0 +1,16 @@
+require "test_helper"
+
+class BusinessMailerTest < ActionMailer::TestCase
+  test "calling welcome enqueues the mailer" do
+    BusinessMailer.with(business:).welcome.deliver_later
+    assert_enqueued_emails 1
```

Comment:
```suggestion
    assert_emails 1 do
      BusinessMailer.with(business:).welcome.deliver_later
    end
```

Reply:
Also, I think I'd rather create/send a notification to better match the developer mailer test.